opentelemetry-appender-tracing/Cargo.toml

```diff
@@ -21,10 +21,12 @@ tracing-subscriber = { version = "0.3", default-features = false, features = ["r
 once_cell = "1.13.0"
 
 [dev-dependencies]
+log = "0.4"
 opentelemetry-stdout = { path = "../opentelemetry-stdout", features = ["logs"] }
+opentelemetry_sdk = { path = "../opentelemetry-sdk", features = ["logs", "testing"]  }
```

Comment:
Having the testing feature of opentelemetry_sdk dependency activated in the dev-dependencies in this section fixes when we run the cargo test command in the opentelemetry-appender-tracing folder.

Also, the opentelemetry-jaeger crate does the same thing.